packages/web/src/components/TeeResults.vue

```diff
@@ -153,6 +170,46 @@ const updateDetailResult = (id: string | number) => {
   programs.setDetailResult(id, props.trackId)
 }
 
+interface costInfos {
```

Comment:
suggestion: By convention, interface's names are normally in PascalCase format  
https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#interfaces